### DISCUSSION TITLE
Misorientation Angle distribution around a specific axis

### DISCUSSION THREAD

**Discussion by d-Wang8** [mtex-toolbox/mtex#1440] [Ask Anything]
Hi,

generally, the misorientation angle distribution given by EBSD software just gives you the smallest misorientation angle and it won't tell you which axis is the rotating axis. Can we ask MTEX to give us something like 'misorientation angle distribution around {111} plane normal' or 'angle between misorientation axis and the nearest {111} pole'?

thanks

**Comment by kilir** [comment]
Hi,

how about:
```
% some input data
cs = crystalSymmetry('432');
mori = orientation.rand(10000,cs,cs);

% 'angle between misorientation axis and the nearest {111} pole'
m_ang_111 = angle(mori.axis ,Miller(1,1,1,cs))

% 'misorientation angle distribution around {111} plane normal'
th = m_ang_111 < 5*degree;
histogram(mori(th).angle/degree)
```

Cheers,
Rüdiger

**Reply by d-Wang8** [reply]
actually I meant to ask:
 the Misorientation distribution histogram only gives you the distribution of 'smallest angles' between any 2 pixels, taken any axes into account. So in my sample, there are three types of twinning and all of them can be described by rotations of multiples of 60° about {111}. So I am wondering is possible to plot the misorientation angle distribution histogram while only considering {111} plane normals as rotation axis.

**Reply by kilir** [reply]
For any arbitrary misorientation?
No. 
Cheers,
Rüdiger
